src/net_processing.cpp

```diff
@@ -3660,17 +3660,8 @@ bool PeerLogicValidation::ProcessMessages(CNode* pfrom, std::atomic<bool>& inter
     // Message size
     unsigned int nMessageSize = msg.m_message_size;
 
-    // Checksum
-    CDataStream& vRecv = msg.m_recv;
-    if (!msg.m_valid_checksum)
-    {
-        LogPrint(BCLog::NET, "%s(%s, %u bytes): CHECKSUM ERROR peer=%d\n", __func__,
```

Comment:
I see in the last commit you add `+    const NodeId m_node_id; // Only for logging`

Maybe this addition could be moved to the first commit? It will be slightly painful due to the conflicts that will need to be solved, but I think it will be worth it

Reply:
The peer id is now present in all of this logging.